### DISCUSSION TITLE
Only deploy minified resources (JS, CSS)

### DISCUSSION THREAD

**Discussion by alexandre-touret** [mmistakes/minimal-mistakes#3263] [Q&A]
Hello,
I deploy minimal mistakes jekyll web site on github pages.
Unfortunately, I have both main.js and main.min.js deployed.
This behaviour is considered as an error for SEO analyzers ( main.js is too long to be downloaded ) 
How could I remove the original JS file ?
I also have the same question the the CSS : how can I minify the CSS and only deploy the minified file.

Thanks

**Comment by iBug** [comment]
There is no `main.js` deployed by default, but there is [`_main.js`](https://github.com/mmistakes/minimal-mistakes/blob/master/assets/js/_main.js) which [is different from the minified one](https://github.com/mmistakes/minimal-mistakes/blob/9ab55b6c5487a770ac18410423e47a6f976dc098/package.json#L30).

Unless you're using a custom build pipeline, these problems should not exist in the first place. CSS is automatically [minified](https://github.com/iBug-web/iBug-web.github.io/tree/master/assets/css) by Jekyll.

**Comment by mmistakes** [comment]
I'll echo @iBug.
`assets/js/main.min.js` comes from the theme and is already minified.
`assets/css/main.css` is also minified by Jekyll's internal Sass support

Not sure what your site's URL is but taking a guess from your GitHub repo I see the CSS and JS on https://blog.touret.info are minified. 

I also see no mention of `main.js` loading in dev tools. Sounds like maybe a false positive with whatever SEO tool you're using. I would take warnings and errors from them with a grain of salt. Like accessibility tools they aren't always accurate and require a manual look sometimes.

![Screen Shot 2021-12-01 at 11 59 08 AM](https://user-images.githubusercontent.com/1376749/144278836-8a014d2f-9aa9-477d-a591-3b498daa0fcb.png)

**Reply by alexandre-touret** [reply]
Hello,
Thanks for your feedback.

Regards
Alexandre
